README.md

```diff
@@ -0,0 +1,175 @@
+<h3><b>Catalog</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 School Library ](#-school-library-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  [🚀 Project Documentation ](#-project-documentation-)
+    - [Video ](#video)
+    - [Diagram ](#diagram)
+  - [💻 Getting Started ](#-getting-started-)
+  - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+  - [Usage](#usage)
+  - [Author ](#author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Catalog <a name="about-project"></a>
+
+**[ Catalog ]** In this project, we are going to create a console app that will help you to keep a record of different types of things you own: books, music albums, movies, and games. Everything will be based on the UML class diagram presented below.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Create and manage data about my things]**
+- **[Add a Book]**
+- **[Add a Game]**  
+- **[Add a Music Album]**
+- **[List all added items]**
+- **[Keep track of books, music albums, movies, and games and other information]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Project Documentation <a name="live-demo"></a>
+
+Please visit our zoom video presentation. Click the [Link](https://loom.com/share/cacf0349f8cb4db49d90f9b9323de431) button.
+

```

Comment:
One of our coding partners (the last in showing up his more valuable part of the code) just had an issue with audio, and it wasn't possible to fix this because it is late at night in its timezone.

Reply:
One of our coding partners (the last in showing up his more valuable part of the code) just had an issue with audio, and it wasn't possible to fix this because it is late at night in its timezone 😢

Reply:
## Dear code reviewer
The re-recording of the video is not going to be posible, because the time diference betwen partners not allow us to connect and do it in time. If you Kindly up your speckers, is posible to hear all participants. Please let us use the current video this time

Thank you very much!